gridstatus/ercot.py

```diff
@@ -1787,6 +1791,65 @@ def _make_bid_curve(df):
 
         return self.parse_doc(df, verbose=verbose)
 
+    @support_date_range("DAY_START")
+    def get_sced_system_lambda(self, date, verbose=False):
+        """Get System lambda of each successful SCED
+
+        5 Minute Publish Interval
+
+        https://github.com/kmax12/gridstatus/issues/269
+
+        Arguments:
+            date (str, datetime): date to get data for
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: A DataFrame
+
+        """
+
+        # just so we can share parse logic
+        kwargs = dict(
+            report_type_id=SCED_SYSTEM_LAMBDA_RTID,
+            date=date,
+            verbose=verbose,
+            extension="csv",
+        )
+
+        if date == "latest":
+            doc = self._get_document(**kwargs)
+            docs = [doc]
+        else:
+            docs = self._get_documents(**kwargs)
+
+        df = self._handle_sced_system_lambda(docs)
```

Comment:
tests arent passing because this is missing the required `verbose` parameter. let me know if you need help setting up tests to run. 

ill run it on github action, but unfortunately its current broken due to an unrelated issue of requests to SPP's server not working on github

Reply:
Oh i have no idea how to set up those tests. Help would be nice

Reply:
does spp block the github server or something?

Reply:
verbose added tho

Reply:
start here for tests: https://github.com/kmax12/gridstatus/blob/main/CONTRIBUTING.md#running-tests-and-linting

Reply:
i think the problem with SPP has to do with SSL cert configuration on the docker image used for the github action. i need to investigate more